base/commandLine.cpp

```diff
@@ -1669,6 +1669,73 @@ bool processSettings(Common::String &command, Common::StringMap &settings, Commo
 		}
 	}
 
+	// store all session related settings
+	if (settings.contains("savepath")) {
```

Comment:
I'd recommend either refactoring it into a method or use macro, e.g. something like:

```C++
storeSessionSetting("savepath");
storeSessionSetting("themepath");
```
etc